src/consensus/pbft_manager.cpp

```diff
@@ -1852,7 +1852,7 @@ void PbftManager::syncBlockQueuePush(SyncBlock &&block, dev::p2p::NodeID const &
 }
 
 void PbftManager::clearSyncBlockQueue() {
-  sync_period_ = 0;
+  sync_period_ = pbft_chain_->getPbftChainSize();
```

Comment:
This seems unnecessary as `pbft_chain_->getPbftChainSize()` would be returned from `pbftSyncingPeriod()` in `syncBlockQueuePush` anyway...